components/container-cards/contact-card/contact-card.hbs

```diff
@@ -0,0 +1,216 @@
+<div class="container">
+  <div class="row">
+    <div class="col-12">
+      <h1>Sizes & variations</h1>
+      <h2 style="font-size: 14px;	font-weight: bold; color: #6D6F71;">WITH IMAGE</h2>
+    </div>
+
+    <div class="col-6">
+      <h4 style="font-size: 14px;	font-weight: bold; color: #6D6F71;">Large</h4>
```

Comment:
you could just use the typography classes instead.

Reply:
I have done the corrections based on your feedback, now using:
 <div class="msds-text-display-3">Sizes & variations</div>
 <div class="msds-text-body-1">WITH IMAGE</div>